src/features/events/components/groupedEventsCard.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import moment from 'moment'
+import { FlexColumn } from '../../../components/grid'
+import { media } from '../../../theme/media'
+import constants from '../../../constants'
+import EventListingCard from './eventListingCard'
+
+const DateGroupHeading = styled.h2`
+  margin: 1rem 0;
+
+  ${media.tablet`
+    display: none;
+  `};
+`
+
+const GroupedEventsCards = ({ event, index, events, toLoad }) => {
+  let header
+  const longDayOfMonth = 'dddd D MMM'
+
+  if (index === 0) {
+    header = moment(event.node.startTime).format(longDayOfMonth)
+  } else {
+    const startDate = moment(event.node.startTime).format(constants.dateFormat)
+    const prevStartDate = moment(events[index - 1].node.startTime).format(
+      constants.dateFormat
+    )
+
+    if (startDate !== prevStartDate) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4658697)